ultraleap/TouchFree
# TF-358: Welcome Screen for Web-based settings UI

# Checklist:
- [x] Test plan updated
- [ ] Developer Testing: Key item tests 
- [ ] Developer Testing: Execution of the updated test plan (changed items for this work only) 
- [ ] Product Owner Review (Optional) 
- [x] Update Release Notes on the JIRA ticket for harvest at release time

# Testing Procedures:
* Install the build of TouchFree, a build of tracking, and connect a camera
* Navigate to http://localhost:9739/settings/index.html
* Observe the Welcome Screen
* Observe the status icons in the top left (expect both dots to be green)
* Disconnect the camera, and wait 5 seconds
    * The dot for the camera should go red
* Open "Services" in windows, find and stop the Ultraleap Tracking Service
    * The dot for the Service (the hand icon) should go red
* Reconnect the camera, then restart the Tracking Service
    * Both dots should go green again
